persimmon/website/views.py

```diff
@@ -133,3 +144,49 @@ def get_my_accounts(request):
         'balance': account.balance,
         'approval_status': account.approval_status,
     } for account in accounts]
+
+
+@api_function
+def approve_credit_debit_funds(request, account_number: int, approved: bool, credittype0debittype1: int):
+    current_user(request, required_auth=EmployeeLevel.TELLER)
+    try:
+        pendingtransactions = BankStatements \
+            .objects.filter(bankAccountId=account_number, approval_status=ApprovalStatus.PENDING)
+    except BankStatements.DoesNotExist as exc:
```

Comment:
will `filter` raise an exception if there are no matches or will it just return an empty query?

Reply:
I will check it.